controllers/v1/assessmentsController.js

```diff
@@ -134,7 +134,7 @@ module.exports = class Assessments {
                 let themeCriterias = new Array
 
                 if (eachTheme.children) {
-                    themeCriterias = controllers.schoolsController.getCriteriaIds(eachTheme.children)
+                    themeCriterias = gen.utils.getCriteriaIds(eachTheme.children)
```

Comment:
We can move the entire theme loop in a common function as well instead of calling the same function for each theme as done in import/export controller.